Projectoid.js

```diff
@@ -0,0 +1,313 @@
+//The Global Variables
+let libPrefix = "projectoid";
+let API_URL = "https://api.projectoid.site/v1";
+
+// Function to convert JSON to URL query string
+function jTQS(jsonData) {
+  let queryString = Object.keys(jsonData)
+    .map(function (key) {
+      return encodeURIComponent(key) + "=" + encodeURIComponent(jsonData[key]);
+    })
+    .join("&");
+  return queryString;
+}
+
+/**
+ * apiCall() - Makes an API call to Projectoid.
+ *
+ * @since 1.0.0
+ *
+ * @param {object} options - The API call options.
+ */
+function apiCall(options) {
+  if (!options) {
+    throw libPrefix + ": apiCall: options not found";
+  }
+  let url = API_URL + "/" + options.path;
+  if (options.method.toLowerCase() === "get") {
+    url += "?" + jTQS(options.body);
+  }
+  let apiData = {
+    url: url,
+    headers: {
+      ...(options.method.toLowerCase() === "get"
+        ? {}
+        : { "Content-Type": "application/json" }),
+    },
+    body: options.body,
+    folow_redirects: true,
```

Comment:
redirect is really needed?

Reply:
yes it is needed.

i don't know why, but sometimes in bb it creates a issue if don't use it.